app/components/verkiezingen/bcsd-voorzitter-alert.js

```diff
@@ -1,130 +1,75 @@
 import Component from '@glimmer/component';
+
 import { tracked } from '@glimmer/tracking';
 import { service } from '@ember/service';
-import { task, timeout } from 'ember-concurrency';
-import { findFirst } from 'frontend-lmb/utils/async-array-functions';
+import { action } from '@ember/object';
+import { A } from '@ember/array';
+
+import { task } from 'ember-concurrency';
+
 import { queryRecord } from 'frontend-lmb/utils/query-record';
 import {
-  MANDAAT_LID_VAST_BUREAU_CODE,
-  MANDAAT_LID_RMW_CODE,
-  MANDAAT_VOORZITTER_BCSD_CODE,
-} from 'frontend-lmb/utils/well-known-uris';
+  MANDAAT_SCHEPEN_CODE_ID,
+  MANDAAT_TOEGEVOEGDE_SCHEPEN_CODE_ID,
+} from 'frontend-lmb/utils/well-known-ids';
 
 export default class VerkiezingenBcsdVoorzitterAlertComponent extends Component {
   @service store;
-  @service installatievergadering;
-
-  @tracked errorMessage = '';
-  @tracked lastRecomputeTime = null;
-
-  constructor() {
-    super(...arguments);
-    this.installatievergadering.forceRecomputeBCSD();
-  }
-
-  get bcsdBestuursorgaanInTijd() {
-    return this.args.bcsdBestuursorgaanInTijd;
-  }
 
-  get bestuursorganenInTijd() {
-    return this.args.bestuursorganenInTijd;
-  }
-
-  // Polling because otherwise changes in the installatievergadering data are not picked up
-  handleErrorMessage = task({ restartable: true }, async () => {
-    if (this.isDestroyed) {
-      return;
-    }
-    if (
-      this.lastRecomputeTime &&
-      this.lastRecomputeTime ===
-        this.installatievergadering.recomputeBCSDNeededTime
-    ) {
-      await timeout(10000);
-      // nothing for now let's try again later
-      this.handleErrorMessage.perform();
-      return;
+  @tracked messages = A();
+
+  isVoorzitterAlsoSchepen = task(async () => {
+    const bcsdMandatarissen = this.args.mandatarissen;
+    const mapping = await Promise.all(
+      bcsdMandatarissen.map(async (mandataris) => {
+        return {
+          mandataris,
+          persoon: await mandataris.isBestuurlijkeAliasVan,
+          isVoorzitter: await (await mandataris.bekleedt).isVoorzitter,
+        };
+      })
+    );
+    const hasVoorzitter = mapping.find(
+      (mapping) => mapping.persoon && mapping.isVoorzitter
+    );
+    if (hasVoorzitter) {
+      const schepen = await this.findMandatarisForOneOfBestuursfunctieCodes(
+        hasVoorzitter.persoon,
+        [MANDAAT_SCHEPEN_CODE_ID, MANDAAT_TOEGEVOEGDE_SCHEPEN_CODE_ID]
```

Comment:
i think burgemeester and aangewezen burgemeester should also be allowed because they just have to be member of vast bureau so the mayor is allowed to do this himself i think

Reply:
indeed: chair of vb (so the mayor) can do this too: https://www.vlaanderen.be/lokaal-bestuur/organisatie-en-werking/bijzonder-comite-voor-de-sociale-dienst

Reply:
Adde burgemeester and aangewezen to the mandaten array , https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/442/commits/f7d945e5a089ce26f3ded2bc1ceae994b244e057